vault-admin-client/src/main/java/com/scality/vaultadmin/impl/VaultAdminImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.scality.vaultadmin.impl;
+
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.scality.vaultclient.dto.CreateAccountRequestDTO;
+import com.scality.vaultclient.dto.CreateAccountResponseDTO;
+import com.scality.vaultclient.services.AccountServicesClient;
+import com.scality.vaultclient.services.VaultClientException;
+import okhttp3.*;
+import com.scality.vaultadmin.VaultAdmin;
+
+/**
+ * Vault administrator implementation
+ *
+ * <p>Created by saitharun on 12/16/20.
+ */
+@SuppressWarnings("deprecation")
+public class VaultAdminImpl implements VaultAdmin {
+
+  private final AccountServicesClient vaultAccountclient;
```

Comment:
```suggestion
  private final AccountServicesClient vaultAccountClient;
```

Reply:
Piggybacking on this as I was commenting the same thing.

Reply:
addressed this in the commit [84b8df6](https://github.com/scality/vmware-ose-scality/pull/12/commits/84b8df6c060bad85caaac8b202068fa5f85470e5)